src/components/ebay-notice/README.md

```diff
@@ -19,3 +23,11 @@ Name | Type | Stateful | Description
 `status`  | String | No | "priority" (default), "confirmation" or "information"
 `aria-text` | String | No | adding description for the notice for a11y users
 `heading-level` | String | No| used in case of "page" level notices to specify the heading tag according to the notice's placement
+`dismissible` | Boolean | No | used to specify the dismiss button in page notice
+`hidden` | Boolean | Yes | whether the widget is hidden or not (Note: not supported as initial attribute)
+
+### ebay-notice Events
+
+Event | Description
+--- | ---
+`notice-dismissed` | the page notice was dismissed/closed
```

Comment:
I think this should probably be `notice-dismiss` to be consistent with other command-form verbs in events.

Reply:
okay will change that, I was just thinking that as the event fired describes what already occurred.